### ISSUE TITLE
[Community Review] Pre-discussion: 4th NumbersDAO voting topic community review

### ISSUE THREAD

**Issue by sofiayan0523** [numbersprotocol/community-support#292]
### Overview
Today, we embark on a journey of exploration and discussion surrounding an important decision that lies at the heart of our decentralized ecosystem. We are here to deliberate on the question of whether Numbers should open up its validator set to the wider community.

At Numbers Protocol, we deeply value our community and the principles of decentralization. Our shared vision is to build a robust and resilient network that empowers its participants, enables secure transactions, and fosters trust. The selection of validators plays a critical role in achieving these goals within a proof-of-stake blockchain. Currently, our validator slots are permissions and limited to Numbers partners. However, we believe it is time to open up the discussion and consider the advantages and challenges of embracing a more inclusive approach.

This article will examine the pros and cons of allowing access to the current permission Numbers blockchain validator set and implementing a delegated proof-of-stake (PoS) mechanism, similar to other prominent layer-1 blockchains. We aim to provide you with an impartial analysis that equips you with the necessary information to understand the dialogue that will take place during the upcoming DAO meeting. Remember, this article is not meant to sway your opinion but rather to foster dialogue and broaden our collective understanding.

So, let us delve into the intricacies of decentralizing the validator set and explore its potential impact on our community, protocol, and Numbers' future.

### Decentralizing the validator set
Before diving into the discussion around decentralizing the validator set, first, we will outline what a validators role is within a proof-of-stake blockchain:

_In Proof of Stake blockchains, validators are selected to produce the next block based on their stake in the network token. These validators will stake some of NUM tokens and run software that allows them to produce and approve blocks. Although often designed with random functions to prevent a front-running consensus, a larger amount staked by a validator could give them a higher chance of producing the next block. Proposed blocks by validators are then propagated to the rest of the set, who verify and add the approved block to the blockchain._ 

In the current implementation of the Numbers Blockchain, these validator slots are permissions and only accessible to Numbers partners. 

Community and decentralization are core values of Numbers Protocol, and with these values in mind, we have decided to open up the discussion around allowing access to the currently permissioned Numbers blockchain validator set and implementing a delegated proof-of-stake (PoS) mechanism, similar to other popular layer-1 blockchains.

This community article aims to outline the pros and cons of opening the validator set to the community:

**Pros**
- Opening the validator set is in-line with the core values of Numbers Protocol, which are community and decentralization
- Allows for NUM stakers to delegate their stake to a validator of their choice to share in validation rewards
- Adds additional utility for the NUM token, as this token will be required to be staked and risks slashing in the case of harmful behavior
- Allows for additional rewards for those community members who want to validate the Numbers Blockchain
- Increases decentralization and attack resilience in the event of a malicious validator attacking the network

**Cons**
- Current validators are permissioned strategic partners of Numbers Protocol, which includes AVAX. By opening the validator set, this opens the door to higher chances of bad actors. Once implemented this action is irreversible, as the validator set will become permissionless and there is no way to remove all new validators.
- Opening the validator set will require technical resources and costs from Numbers Protocol revenues
- Running a validator will require technical expertise on the community members part to run effectively and without downtime in addition to a sizeable stake in NUM

Numbers is actively working with its partner network to assign more utility to the NUM token and work towards a more decentralized future. This article aims at opening the discussion between the community if a decentralized validator set is something that would like to be explored.

### Conclusion
Ultimately, the decision to decentralize the validator set within Numbers Protocol is one of great significance and should not be taken lightly. As we assess the advantages and challenges of such a step, we must remain steadfast in our commitment to community and decentralization, which lie at the core of our protocol.

By opening up the validator set, we can embrace a more inclusive approach that empowers NUM stakers to delegate their stake, share in validation rewards, and contribute to the robustness of our blockchain. This move enhances the utility of the NUM token and fortifies the network's resilience against malicious attacks. However, we must be mindful of the additional risks that come with decentralization, such as potential bad actors and the technical resources required to support a larger validator set.

Numbers Protocol remains neutral in this discussion. We encourage all stakeholders to engage in open dialogue and share their perspectives.

Together, let us shape the future of Numbers Protocol as we work towards a more decentralized and inclusive ecosystem driven by the collective wisdom of our community. Your voice matters, and we eagerly anticipate the outcome of this crucial decision.

Thank you for being an integral part of the Numbers Protocol journey!

### Entering Community Review
Just like previous DAO meetings topics, this Numbers Governance Proposal will undergo community review until June 20. During this time, we encourage the Numbers community to share your thoughts and concerns on GitHub issues and discuss this proposal openly on our many social channels. We look forward to hearing your feedback!

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1203248692432071/1204757790001833) by [Unito](https://www.unito.io)

**Comment by Doc-Goodluck** [comment]
For me, my app works perfectly well. I'm using Redmi 8 so I have no problem although I noticed that the some finds it hard downloading/using the CaptureApp due to the type of phone they are using. 

My humble suggestion is let the team try to make the app accessable by all android phones

**Comment by ethanwu155** [comment]
> For me, my app works perfectly well. I'm using Redmi 8 so I have no problem although I noticed that the some finds it hard downloading/using the CaptureApp due to the type of phone they are using.
> 
> My humble suggestion is let the team try to make the app accessable by all android phones

@Doc-Goodluck i believe this comment was placed in teh wrong GitHub Issue thread. This thread is the Community Review for the June Numbers DAO meeting.

The topic for this meeting is: Should Numbers open up mainnet validators to the community.

If you have time, we would love to hear your thoughts and opinions on this topic. If you have questions feel free to share them as well.

**Comment by TylerDylan** [comment]
Based on the provided information, it seems that opening up the validator set and implementing delegated proof-of-stake (PoS) could be a favorable decision for Numbers. Here are the reasons supporting this stance:

Pros:

Alignment with core values: Opening the validator set and embracing decentralization aligns with the core values of Numbers Protocol, which prioritize community involvement and decentralization. It allows for greater participation and empowers community members to contribute to the network's validation process.

Delegation and rewards: Allowing NUM token holders to delegate their stake to validators of their choice introduces a mechanism for sharing in validation rewards. This incentivizes token holders to actively participate and stake their NUM tokens, adding utility to the token itself.

Increased utility and risk mitigation: Requiring the staking of NUM tokens and the possibility of slashing in the case of harmful behavior adds utility to the token. It also acts as a deterrent against malicious actions, as validators risk losing their staked tokens if they engage in harmful behavior. This can contribute to a more secure and trustworthy network.

Decentralization and resilience: Opening the validator set to the wider community increases decentralization and enhances the network's resilience against malicious attacks. By distributing the validation power among a larger set of validators, the network becomes less susceptible to control by a single entity or a small group of validators.

Cons:

Potential for bad actors: Opening up the validator set does introduce the possibility of bad actors entering the network. Currently, the validator set consists of permissioned strategic partners who have undergone vetting. With open access, there is a higher chance of malicious validators being included. This risk needs to be carefully managed to ensure the network's security.

Technical resources and costs: Opening the validator set and supporting a larger number of validators would require additional technical resources and incur costs for Numbers Protocol. The infrastructure and support needed for an expanded validator set should be carefully considered to maintain a reliable and robust network.

In conclusion, opening up the validator set and implementing delegated PoS appears to have several benefits for Numbers Protocol. It aligns with the core values of community and decentralization, introduces delegation and rewards, increases utility and risk mitigation, and enhances network decentralization and resilience. However, the potential risks of bad actors and the technical requirements and costs should be thoroughly addressed. It is essential for the community to engage in open dialogue, share perspectives, and carefully consider the implications before making a final decision.

**Comment by Nezukokamado11** [comment]
The decision to open up the validator set and implement delegated proof-of-stake (PoS) in Numbers Protocol is a crucial one that requires careful consideration. Here is my analysis of the topic:

Pros:

1. Alignment with core values: Opening up the validator set and embracing decentralization is in line with the core values of Numbers Protocol, which prioritize community involvement and decentralization. It allows for greater participation and empowers community members to contribute to the network's validation process.

2. Delegation and rewards: Allowing NUM token holders to delegate their stake to validators of their choice introduces a mechanism for sharing in validation rewards. This incentivizes token holders to actively participate and stake their NUM tokens, adding utility to the token itself.

3. Increased utility and risk mitigation: Requiring the staking of NUM tokens and the possibility of slashing in the case of harmful behavior adds utility to the token. It also acts as a deterrent against malicious actions, as validators risk losing their staked tokens if they engage in harmful behavior. This can contribute to a more secure and trustworthy network.

4. Decentralization and resilience: Opening the validator set to the wider community increases decentralization and enhances the network's resilience against malicious attacks. By distributing the validation power among a larger set of validators, the network becomes less susceptible to control by a single entity or a small group of validators.

Cons:

1. Potential for bad actors: Opening up the validator set does introduce the possibility of bad actors entering the network. Currently, the validator set consists of permissioned strategic partners who have undergone vetting. With open access, there is a higher chance of malicious validators being included. This risk needs to be carefully managed to ensure the network's security.

2. Technical resources and costs: Opening the validator set and supporting a larger number of validators would require additional technical resources and incur costs for Numbers Protocol. The infrastructure and support needed for an expanded validator set should be carefully considered to maintain a reliable and robust network.

In conclusion, opening up the validator set and implementing delegated PoS in Numbers Protocol has several advantages, including alignment with core values, delegation and rewards, increased utility and risk mitigation, and enhanced decentralization and resilience. However, the potential risks of bad actors and the technical requirements and costs should be thoroughly addressed. It is important for the community to engage in open dialogue, share perspectives, and carefully consider the implications before making a final decision.

**Comment by DexstarJay** [comment]
I was having issues with launching Capture app on My phone but as soon as I update My phone to the latest version, everything is working perfectly now. I love My Capture App.😊😎🤲🏻

**Comment by fredlex** [comment]
In my opinion.. opening the validator set to a wider Numbers community while implementing delegated proof of stake is seen from a positive light as being beneficial improving inter relationship and trust building among members of the community. However, critical decisions needs to be made. 
 
Here are few advantages of opening the validator set:

Opening the validator set is in-line with the core values of Numbers Protocol, which are community and decentralization
Allows for NUM stakers to delegate their stake to a validator of their choice to share in validation rewards
2. Staking reward: Adds additional utility for the NUM token, as this token will be required to be staked and risks slashing in the case of harmful behavior
Allows for additional rewards for those community members who want to validate the Numbers Blockchain

Cons..
Opening of the validator set would require a robust technical and financial input to present and efficient staking ecosystem

Similarly an expected outcome of potential bad actors should not be ruled out as this may in turn reduce trust level..

**Comment by Marybeth-Nwachukwu** [comment]
The question of whether Numbers should open up the validator set and implement delegated Proof of Stake (PoS) is an important one, and it's worth considering the potential benefits and challenges associated with such a decision.

Pros of Opening up the Validator Set:
1. Decentralization: Opening up the validator set can promote decentralization by allowing more participants to contribute to the network's consensus mechanism. This can enhance the network's security and resilience against attacks.
2. Increased Participation: Allowing more validators to join the network encourages greater participation from the community. It can attract a wider range of stakeholders who are interested in supporting the Numbers protocol, leading to a more diverse and robust ecosystem.

Pros of Implementing Delegated PoS:
1. Accessibility: Delegated PoS can make participation in the consensus process more accessible to individuals who may not have the technical expertise or resources to run their own validator nodes. Delegating stake to trusted validators allows them to represent the interests of smaller stakeholders.
2. Efficiency: Delegated PoS can increase the efficiency of the consensus mechanism by allowing token holders to delegate their voting power to experienced validators. This can reduce the computational and energy requirements compared to traditional PoW consensus algorithms.

Challenges and Considerations:
1. Centralization Risks: Opening up the validator set and implementing delegated PoS should be carefully designed to avoid centralization risks. Measures should be put in place to prevent a small number of validators from gaining excessive control over the network.
2. Governance Mechanisms: Effective governance mechanisms should be established to ensure that decisions related to validator set composition and protocol upgrades are made in a transparent and inclusive manner.

**Comment by Mahabirgope** [comment]
Numbers should open up the validator set and implement delegated PoS:

Pros:

Community involvement and decentralization.
Delegation allows token holders to earn rewards.
Increased utility and risk mitigation.
Improved decentralization and network resilience.
Cons:

Potential for bad actors.
Additional technical resources and costs.

Overall, the community review  that opening up the validator set and implementing delegated PoS brings benefits such as community participation, rewards, and improved network security, but it also introduces risks

**Comment by 4djc** [comment]
My app works flawlessly for me. I'm using a Samsung Galaxy s8+, so I have no issues, however I've seen on my iOS  I have difficulty using the CaptureApp maybe due to its old iOS 7+

Let the dev strive to make the app accessible to all iphones and android phones that are not accessible, in my honest opinion.

**Comment by TylerDylan** [comment]
The provided information suggests that Numbers Protocol should consider opening up its validator set and implementing delegated proof-of-stake (PoS). This decision aligns with the protocol's core values of community involvement and decentralization. Some advantages of this approach include greater community participation, delegation of stake, sharing validation rewards, increased utility of tokens, risk mitigation through staking and slashing, and improved network security. However, there are potential drawbacks to consider, such as the possibility of bad actors entering the network and the need for additional technical resources and costs to support a larger set of validators. It is crucial for the community to engage in open dialogue and carefully evaluate the pros and cons before reaching a final decision.

**Comment by fredlex** [comment]
implementing POS. (Prof of stake) and Opening of the validator set will go a long way in increasing the utility of the numbers protocol native token hence leading to increase in community trust and growth in the numbers ecosystem. This step is in alignment with the core value for the community. In as much as this sounds interesting, there could be some reasons to be worried, such as potential risk of bad actors and the need for financial and technical involvement. 

Contribute: I think a security protection technology be implemented to protect the entire community and certain criteria to be passed before being acknowledged as a validator. This will creat a smooth environment and out bad actors in check ✅

**Comment by Marybeth-Nwachukwu** [comment]
Implementing delegated PoS can provide advantages such as scalability, energy efficiency, and faster transaction confirmations. It also encourages wider participation and engagement from token holders, as they have a say in the consensus process through their voting power. However, it's worth noting that DPoS systems can potentially introduce certain risks, such as the concentration of power in the hands of a small number of delegates or the potential for collusion among them. 

The fact that Numbers Protocol should open up the validator set and implement delegated PoS depends on various factors. It's essential for the project team to carefully consider the goals, values, and specific requirements of their network. They should assess the potential benefits and drawbacks of such a change and consider the impact on security, decentralization, governance, and network performance.

Additionally, it would be crucial for the Numbers project team to engage with their community and stakeholders, seek their input and feedback, and conduct thorough research and analysis before making any significant changes to the consensus mechanism. Ultimately, the decision should align with the project's long-term vision and goals.

**Comment by ethanwu155** [comment]
> I was having issues with launching Capture app on My phone but as soon as I update My phone to the latest version, everything is working perfectly now. I love My Capture App.😊😎🤲🏻

I'm glad your Capture App is working now and thanks so much for your kind words. However this is the community review >.<

Maybe you can take some time to review the topic and let us know your thoughts. Love to know what you think. If you have any questions about the topic feel free to leave a question as well :)

**Comment by Nezukokamado11** [comment]
While I understand the potential benefits, we should also consider the potential risks. Opening the validator set might increase the chance of bad actors infiltrating the network. We need to have a robust vetting process in place to mitigate this risk.

**Comment by Mahabirgope** [comment]
The Numbers Protocol is advised to consider implementing delegated proof-of-stake (PoS) and opening its validator set to align with its values of community involvement and decentralization. This approach offers several benefits, including increased community participation, stake delegation, shared rewards, token utility, risk management, and enhanced network security. However, potential drawbacks exist, such as the risk of malicious actors joining the network and the need for additional technical resources and costs. Therefore, it is important for the community to engage in open discussions and thoroughly evaluate the advantages and disadvantages before making a final decision.

**Comment by nzubezion** [comment]
Yes! Opening up the validator set and implementing delegated PoS for Numbers Protocol is a step towards decentralization, aligning with core values. It allows stake delegation, adds utility to NUM token, and enhances network security.

**Comment by Kristeeyo** [comment]
It's really commendable that the team is looking forward to include the community in its validator roles for more decentralisation

As stated in the stated in the article, there are pros and cons. The delegated proof of stake is an adoptable idea but validator roles should be left for the Numbers partners to be able to keep track and do away with "bad actors"

Through the delegated PoS, community members will still have a voice through their staked tokens and will get to secure rewards for themselves. 
That's the highest community inclusion should go, the community members get the freedom to choose their validators 

Asides bad actors, the technological expertise is another issue the team should look into closely. 
Do these community members have the necessary tools?
Are they conversant with validator roles?

These are questions that should be asked

Delegated proof of stake is a yes but community inclusion in validator roles is a no

**Comment by 4djc** [comment]
In my viewpoint, exposing the validator set to a larger Numbers community while adopting delegated proof of stake is viewed positively as enhancing inter-community relationships and trust development. However, important decisions must be taken.

Here are a few benefits to exposing the validator set:

Opening the validator set is consistent with Numbers Protocol's key objectives of community and decentralization.
Allows NUM stakeholders to delegate their stake to a validator of their choice in order to participate in validation incentives.
Staking reward: Adds additional utility for the NUM token, as this token will be required to be staked and risks slashing in the case of harmful behavior
Allows for additional rewards for those community members who want to validate the Numbers Blockchain

Cons..
Opening of the validator set would require a robust technical and financial input to present and efficient staking ecosystem

Similarly an expected outcome of potential bad actors should not be ruled out as this may in turn reduce trust level..

**Comment by Evagols** [comment]
I feel Numbers opening up mainnet validators to the community has its merits and demerits. 

The merits being that it can increase the utility for the $NUM token and also give members of the community a voice through their staked tokens, thereby enabling them secure rewards for themselves..

The demerits being that there is a possibility of bad actors entering the network if the validator sets are opened.

**Comment by ElieTrades** [comment]
The question is, is it necessary to implement the delegated PoS mechanism? Judging from how most ecosystems using DPoS experience lots of bad actor stories, I suggest we stick to what is more secure. While DPoS offers advantages in terms of speed and scalability, it is important to consider the potential risks associated with it, including the possibility of bad actors and centralization concerns.

If the focus is on security and minimizing risks, sticking to a more secure consensus mechanism, might be a reasonable choice.

**Comment by Wiscobus123** [comment]
In my viewpoint, exposing the validator set to a larger Numbers community while adopting delegated proof of stake is viewed positively as enhancing inter-community relationships and trust development. 
The merits being that it can increase the utility for the $NUM token and also give members of the community a voice through their staked tokens, thereby enabling them secure rewards for themselves.
Staking reward: Adds additional utility for the NUM token, as this token will be required to be staked and risks slashing in the case of harmful behavior

**Comment by TylerDylan** [comment]
Based on the information provided, it seems favorable for Numbers to consider opening up the validator set and implementing delegated proof-of-stake (PoS). Here are the reasons supporting this viewpoint:

Advantages:

Alignment with core values: The decision to open the validator set and embrace decentralization is in line with Numbers Protocol's core values, which prioritize community involvement and decentralization. It enables greater participation and empowers community members to contribute to the network's validation process.

Delegation and rewards: Allowing NUM token holders to delegate their stake to validators of their choice introduces a mechanism for sharing validation rewards. This incentivizes token holders to actively participate and stake their NUM tokens, increasing the token's utility.

Increased utility and risk mitigation: Requiring the staking of NUM tokens and the possibility of slashing in case of harmful behavior adds utility to the token. It also acts as a deterrent against malicious actions since validators risk losing their staked tokens if they engage in harmful behavior. This promotes a more secure and trustworthy network.

Decentralization and resilience: Opening the validator set to a wider community enhances decentralization and strengthens the network's resilience against malicious attacks. By distributing validation power among a larger set of validators, the network becomes less susceptible to control by a single entity or small group.

Disadvantages:

Potential for bad actors: Opening up the validator set introduces the possibility of malicious actors joining the network. Currently, the validator set consists of vetted permissioned strategic partners. With open access, the risk of including malicious validators increases. This risk must be carefully managed to ensure network security.

Technical resources and costs: Opening the validator set and supporting a larger number of validators requires additional technical resources and incurs costs for Numbers Protocol. The infrastructure and support needed for an expanded validator set should be considered to maintain a reliable and robust network.

In conclusion, implementing delegated PoS and opening up the validator set offers several benefits for Numbers Protocol. It aligns with their core values, introduces delegation and rewards, increases utility and risk mitigation, and enhances network decentralization and resilience. However, the potential risks of bad actors and the technical requirements and costs need thorough consideration. It is crucial for the community to engage in open dialogue, share perspectives, and carefully evaluate the implications before reaching a final decision.

**Comment by fredlex** [comment]
Implementing the proof of stake POS For delegates, opening of the validator set to a wider Community helps to build the interrelationship among community members hence creating an improved utility for the $NUM token since this makes community members benefit from the profit of staking being us closers to everything decentralization stands for.  

Proper measures should be taken to eradicate any form of bad actions from validators to keep the community completely secured. ..

**Comment by Mahabirgope** [comment]
Decentralization and resilience: Opening the validator set to a wider community enhances decentralization and strengthens the network's resilience against malicious attacks. By distributing validation power among a larger set of validators, the network becomes less susceptible to control by a single entity or small group.

However, there are certain disadvantages to consider when opening up the validator set:

1. Potential for bad actors: Allowing open access to the validator set introduces the possibility of malicious actors joining the network. Currently, the validator set consists of vetted permissioned strategic partners. With open access, the risk of including malicious validators increases. To maintain network security, it is crucial to carefully manage this risk and implement measures to identify and mitigate bad actors.

2. Technical resources and costs: Opening the validator set and supporting a larger number of validators requires additional technical resources and incurs costs for Numbers Protocol. The infrastructure and support needed for an expanded validator set should be considered to ensure a reliable and robust network. Adequate resources must be allocated to maintain the necessary technical infrastructure and provide support to validators.

Considering the advantages, here are the reasons supporting the viewpoint of opening up the validator set and implementing delegated proof-of-stake (PoS) for Numbers Protocol:

1. Alignment with core values: Opening the validator set and embracing decentralization aligns with Numbers Protocol's core values, which prioritize community involvement and decentralization. It enables greater participation and empowers community members to contribute to the network's validation process, fostering a sense of ownership and engagement.

2. Delegation and rewards: Allowing NUM token holders to delegate their stake to validators of their choice introduces a mechanism for sharing validation rewards. This incentivizes token holders to actively participate and stake their NUM tokens, increasing the token's utility. Delegation also enables token holders with limited technical expertise to still participate and contribute to the network's security.

3. Increased utility and risk mitigation: Requiring the staking of NUM tokens and the possibility of slashing in case of harmful behavior adds utility to the token. It also acts as a deterrent against malicious actions since validators risk losing their staked tokens if they engage in harmful behavior. This promotes a more secure and trustworthy network by aligning the incentives of validators with the network's overall health.

**Comment by Marybeth-Nwachukwu** [comment]
Opening up the validator set and introducing delegated PoS can have several benefits for Numbers. By expanding the validator set, Numbers can increase decentralization, as more participants will have the opportunity to contribute to the network's consensus. This can enhance the overall security and resilience of the blockchain.

Implementing delegated PoS allows token holders to delegate their stake to trusted validators who can represent their interests and participate in the consensus process on their behalf. This approach promotes engagement and participation from a broader community, even those with limited technical knowledge or resources to run their own validators. It can also incentivize token holders to stake their assets, which helps secure the network and maintain its stability.

**Comment by Nezukokamado11** [comment]
Decentralization and resilience: Opening the validator set to the wider community increases decentralization and enhances the network's resilience against malicious attacks. By distributing the validation power among a larger set of validators, the network becomes less susceptible to control by a single entity or a small group of validators.

**Comment by 4djc** [comment]
According to the available data, Numbers Protocol ought to think about expanding its validator network and introducing delegated proof-of-stake (PoS). The protocol's fundamental principles of decentralization and participation of the community are supported by this choice. Greater community involvement, stake delegation, shared validation rewards, enhanced token utility, risk reduction via staking and slashing, and improved network security are some benefits of this strategy. However, there may be disadvantages to take into account, such as the ability for malicious actors to infiltrate the network and the requirement for more technical resources and expenses to support a higher number of validators. Before making a choice, the community must hold an honest discussion and thoroughly weigh the advantages and disadvantages.

**Comment by TylerDylan** [comment]
This suggests that opening up the validator set and adopting delegated proof-of-stake (PoS) would be advantageous for Numbers Protocol. This decision aligns with the protocol's core values of community involvement and decentralization. By allowing token holders to delegate their stake to validators, it introduces a mechanism for sharing validation rewards and encourages active participation. Requiring the staking of tokens adds utility and mitigates risks by deterring malicious behavior through the possibility of slashing. Moreover, decentralizing the validator set enhances the network's security and resilience against attacks.

However, there are potential drawbacks to consider. Opening up the validator set increases the risk of bad actors entering the network, as compared to the current vetted strategic partners. Managing this risk is crucial to maintain network security. Additionally, expanding the validator set requires additional technical resources and incurs costs for Numbers Protocol. Adequate infrastructure and support must be provided to ensure a reliable and robust network.

In conclusion, opening up the validator set and implementing delegated PoS can bring numerous benefits to Numbers Protocol, including alignment with core values, delegation and rewards, increased utility and risk mitigation, and enhanced decentralization and resilience. However, it is essential to address the risks of bad actors and carefully consider the technical requirements and costs. Community engagement, open dialogue, and thoughtful consideration are necessary before making a final decision.

**Comment by RisingStar05** [comment]
I suggest we stick to the previous mechanism simply because it’s less risky that way. Your Validators are currently your partners who of course you have a lot of trust in. How much do we know about each community member? It's highly unlikely that there will be a 50% chance of bad actors but still, I do feel DPoS is not an option.

**Comment by cryptomate7863** [comment]
Dear Numbers Protocol Community,

Thank you for initiating this important discussion regarding the decentralization of the validator set and the implementation of delegated proof-of-stake (PoS) within Numbers Protocol. I believe that considering this decision is crucial for the continued growth and evolution of our decentralized ecosystem.

After carefully reviewing the pros and cons outlined in the article, I would like to share my thoughts on this matter.

Firstly, I fully support the idea of opening up the validator set to the wider community. It aligns perfectly with the core values of Numbers Protocol, emphasizing the importance of community participation and decentralization. By allowing NUM token holders to delegate their stake to validators of their choice, we empower the community and provide them with an opportunity to actively contribute to the validation process. This increased participation not only enhances the security and resilience of our blockchain but also fosters a sense of ownership and engagement among NUM token holders.

Furthermore, opening the validator set adds significant utility to the NUM token. By requiring staking of NUM tokens for participation in the validation process, we create a strong incentive for community members to hold and stake their tokens, thereby promoting the growth and stability of our ecosystem. This increased demand for NUM tokens can have a positive impact on their value and liquidity.

I also appreciate the emphasis on network resilience and protection against malicious attacks. Opening up the validator set and expanding the pool of validators significantly reduces the risk of a single validator becoming compromised and potentially harming the network. A decentralized validator set ensures that the power is distributed among many participants, making it more difficult for any malicious actor to manipulate the system.

However, I acknowledge the potential challenges and concerns associated with this decision. The current permissioned validator set, which consists of strategic partners, has likely been carefully vetted and trusted. Opening up the validator set does introduce the possibility of bad actors entering the network. To address this concern, it will be essential to establish a robust vetting and governance process for new validators, ensuring that they meet certain criteria and adhere to strict guidelines. Transparency and accountability should be prioritized to maintain the integrity of the network.

Additionally, it is important to recognize the technical resources and costs required to support a larger validator set. As the number of validators increases, the infrastructure must be able to handle the increased load and maintain a high level of performance and reliability. Numbers Protocol should carefully assess these requirements and allocate the necessary resources to support the expanded validator set effectively.

In conclusion, I firmly believe that opening up the validator set and implementing delegated PoS within Numbers Protocol is a progressive step forward. It aligns with our core values of community and decentralization, enhances the utility of the NUM token, and strengthens the security and resilience of our network. However, it is crucial that we address the associated challenges, such as vetting new validators and allocating resources effectively.

I encourage all members of the Numbers Protocol community to actively participate in this discussion and share their perspectives. Let us collaborate and shape the future of our protocol in a manner that reflects the collective wisdom and aspirations of our community.

Thank you for providing us with this opportunity to voice our opinions and contribute to the decision-making process.

Sincerely,
Mohammed saqib AKA Cryptomate :)

**Comment by Mahabirgope** [comment]
Opening up the validator set and adopting delegated proof-of-stake (PoS) would be advantageous for Numbers Protocol. It aligns with the protocol's core values of community involvement and decentralization, while introducing mechanisms for sharing validation rewards and encouraging active participation. Requiring token staking adds utility and deters malicious behavior. Decentralizing the validator set enhances security and resilience.

However, there are potential drawbacks, such as the risk of bad actors entering the network. Managing this risk is crucial. Expanding the validator set requires additional technical resources and costs. Sufficient infrastructure and support must be provided.

In conclusion, opening up the validator set and implementing delegated PoS offer benefits like alignment with core values, delegation and rewards, increased utility, risk mitigation, and enhanced decentralization. However, addressing the risk of bad actors and considering technical requirements and costs is important. Community engagement, open dialogue, and careful consideration are necessary before finalizing the decision.

**Comment by Nezukokamado11** [comment]
Based on the available data, Numbers Protocol should consider expanding its validator network and implementing delegated proof-of-stake (PoS). This decision aligns with the protocol's core principles of decentralization and community participation. By adopting this approach, the protocol can achieve several benefits, including increased community engagement, stake delegation, shared validation rewards, improved token utility, reduced risk through staking and slashing, and enhanced network security. Nevertheless, it is important to consider potential drawbacks, such as the possibility of malicious actors infiltrating the network and the need for additional technical resources and expenses to support a larger number of validators. Before making a decision, the community should engage in an open and thorough discussion to carefully assess the advantages and disadvantages.

**Comment by 4djc** [comment]
The advantages of permitting community participation in DPoS outweigh the disadvantages. It is consistent with the goals of the project to nominate validators, and by this I mean actively involve the community in the process.

To earn the community's confidence, the validators, in my opinion, must be well-known inside the community and have a history of sustained membership. You can choose at random the validators with the most staked NUM and then share the list with the community since the weight of validators is based on the quantity of staking tokens. Allow the neighborhood time to conduct individual background checks and determine whose Validators they were

**Comment by Marybeth-Nwachukwu** [comment]
The decision of whether the Numbers Protocol should look to open up the validator set and implement delegated Proof of Stake (PoS) depends on several factors. Here are some key considerations to keep in mind:

1. Decentralization: Opening up the validator set and implementing delegated PoS can potentially increase the decentralization of the Numbers Protocol. By allowing a larger number of validators to participate, it reduces the concentration of power among a few entities, making the network more resilient and less vulnerable to attacks or censorship.

2. Security: Implementing delegated PoS could enhance the security of the protocol. By involving more participants in the consensus process, it becomes more difficult for any single entity or group to compromise the network. However, it's important to ensure that the delegation mechanism is designed securely to avoid potential risks or vulnerabilities.

3. Scalability: Opening up the validator set can help improve the scalability of the Numbers Protocol. If the current validator set is unable to handle the transaction load efficiently, expanding the set can distribute the workload and enable faster processing. Delegated PoS can also allow for more efficient block validation and consensus, further enhancing scalability.

4. Governance and Participation: Delegated PoS can encourage broader participation in the protocol's governance. Token holders can delegate their stake to validators they trust, providing them with the ability to participate in consensus and decision-making. This can promote a more inclusive and democratic governance model, giving stakeholders a voice in shaping the protocol's future.

5. Trade-offs: There are trade-offs to consider when opening up the validator set and implementing delegated PoS. It can introduce complexities in terms of managing the delegation process, potential risks of centralization through large delegated stakes, and challenges in maintaining security and performance. These trade-offs should be carefully evaluated and mitigated to ensure the overall effectiveness and integrity of the protocol.

Ultimately, the decision to open up the validator set and implement delegated PoS should be based on a thorough analysis of the specific requirements, goals, and challenges of the Numbers Protocol. It's important to consider the potential benefits in terms of decentralization, security, scalability, and governance, while also addressing any potential drawbacks and ensuring that the chosen mechanisms align with the protocol's objectives. Consulting with domain experts, conducting feasibility studies, and engaging the community in the decision-making process can help in making an informed choice.

**Comment by qqorq** [comment]
> ### Topic: Should Numbers look to open up the validator set and implement delegated PoS?
> ### Overview
> Today, we embark on a journey of exploration and discussion surrounding an important decision that lies at the heart of our decentralized ecosystem. We are here to deliberate on the question of whether Numbers should open up its validator set to the wider community.
> 
> At Numbers Protocol, we deeply value our community and the principles of decentralization. Our shared vision is to build a robust and resilient network that empowers its participants, enables secure transactions, and fosters trust. The selection of validators plays a critical role in achieving these goals within a proof-of-stake blockchain. Currently, our validator slots are permissions and limited to Numbers partners. However, we believe it is time to open up the discussion and consider the advantages and challenges of embracing a more inclusive approach.
> 
> This article will examine the pros and cons of allowing access to the current permission Numbers blockchain validator set and implementing a delegated proof-of-stake (PoS) mechanism, similar to other prominent layer-1 blockchains. We aim to provide you with an impartial analysis that equips you with the necessary information to understand the dialogue that will take place during the upcoming DAO meeting. Remember, this article is not meant to sway your opinion but rather to foster dialogue and broaden our collective understanding.
> 
> So, let us delve into the intricacies of decentralizing the validator set and explore its potential impact on our community, protocol, and Numbers' future.
> 
> ### Decentralizing the validator set
> Before diving into the discussion around decentralizing the validator set, first, we will outline what a validators role is within a proof-of-stake blockchain:
> 
> _In Proof of Stake blockchains, validators are selected to produce the next block based on their stake in the network token. These validators will stake some of NUM tokens and run software that allows them to produce and approve blocks. Although often designed with random functions to prevent a front-running consensus, a larger amount staked by a validator could give them a higher chance of producing the next block. Proposed blocks by validators are then propagated to the rest of the set, who verify and add the approved block to the blockchain._
> 
> In the current implementation of the Numbers Blockchain, these validator slots are permissions and only accessible to Numbers partners.
> 
> Community and decentralization are core values of Numbers Protocol, and with these values in mind, we have decided to open up the discussion around allowing access to the currently permissioned Numbers blockchain validator set and implementing a delegated proof-of-stake (PoS) mechanism, similar to other popular layer-1 blockchains.
> 
> This community article aims to outline the pros and cons of opening the validator set to the community:
> 
> **Pros**
> 
> * Opening the validator set is in-line with the core values of Numbers Protocol, which are community and decentralization
> * Allows for NUM stakers to delegate their stake to a validator of their choice to share in validation rewards
> * Adds additional utility for the NUM token, as this token will be required to be staked and risks slashing in the case of harmful behavior
> * Allows for additional rewards for those community members who want to validate the Numbers Blockchain
> * Increases decentralization and attack resilience in the event of a malicious validator attacking the network
> 
> **Cons**
> 
> * Current validators are permissioned strategic partners of Numbers Protocol, which includes AVA. By opening the validator set, this opens the door to higher chances of bad actors. Once implemented this action is irreversible, as the validator set will become permissionless and there is no way to remove all new validators.
> * Opening the validator set will require technical resources and costs from Numbers Protocol revenues
> * Running a validator will require technical expertise on the community members part to run effectively and without downtime in addition to a sizeable stake in NUM
> 
> Numbers is actively working with its partner network to assign more utility to the NUM token and work towards a more decentralized future. This article aims at opening the discussion between the community if a decentralized validator set is something that would like to be explored.
> 
> ### Conclusion
> Ultimately, the decision to decentralize the validator set within Numbers Protocol is one of great significance and should not be taken lightly. As we assess the advantages and challenges of such a step, we must remain steadfast in our commitment to community and decentralization, which lie at the core of our protocol.
> 
> By opening up the validator set, we can embrace a more inclusive approach that empowers NUM stakers to delegate their stake, share in validation rewards, and contribute to the robustness of our blockchain. This move enhances the utility of the NUM token and fortifies the network's resilience against malicious attacks. However, we must be mindful of the additional risks that come with decentralization, such as potential bad actors and the technical resources required to support a larger validator set.
> 
> Numbers Protocol remains neutral in this discussion. We encourage all stakeholders to engage in open dialogue and share their perspectives.
> 
> Together, let us shape the future of Numbers Protocol as we work towards a more decentralized and inclusive ecosystem driven by the collective wisdom of our community. Your voice matters, and we eagerly anticipate the outcome of this crucial decision.
> 
> Thank you for being an integral part of the Numbers Protocol journey!
> 
> ### Entering Community Review
> Just like previous DAO meetings topics, this Numbers Governance Proposal will undergo community review until June 20. During this time, we encourage the Numbers community to share your thoughts and concerns on GitHub issues and discuss this proposal openly on our many social channels. We look forward to hearing your feedback!
> 
> ┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1203248692432071/1204757790001833) by [Unito](https://www.unito.io)

I personally think to everything said here and pointed out, shouldn't be taken lightly and the decentralisation can go on. From my end I have no negative energy there with this topic. All good

**Comment by Mahabirgope** [comment]
In light of the available data, it is advisable for Numbers Protocol to contemplate expanding its validator network and implementing delegated proof-of-stake (PoS). Such a decision would be in line with the protocol's fundamental principles of decentralization and community involvement. By embracing this approach, the protocol can enjoy numerous benefits, which include heightened community participation, stake delegation, the sharing of validation rewards, improved token utility, reduced risks through staking and slashing mechanisms, and bolstered network security. However, it is crucial to consider potential drawbacks, such as the potential infiltration of malicious actors into the network and the requirement for additional technical resources and expenses to support a larger set of validators. Before reaching a decision, it is imperative for the community to engage in an open and comprehensive discussion in order to carefully evaluate the advantages and disadvantages.

**Comment by TylerDylan** [comment]
Based on the provided information, it seems that opening up the validator set and implementing delegated proof-of-stake (PoS) could be a favorable decision for Numbers. Here are the reasons supporting this stance:

Pros:

Alignment with core values: Opening the validator set and embracing decentralization aligns with the core values of Numbers Protocol, which prioritize community involvement and decentralization. It allows for greater participation and empowers community members to contribute to the network's validation process.

Delegation and rewards: Allowing NUM token holders to delegate their stake to validators of their choice introduces a mechanism for sharing in validation rewards. This incentivizes token holders to actively participate and stake their NUM tokens, adding utility to the token itself.

Increased utility and risk mitigation: Requiring the staking of NUM tokens and the possibility of slashing in the case of harmful behavior adds utility to the token. It also acts as a deterrent against malicious actions, as validators risk losing their staked tokens if they engage in harmful behavior. This can contribute to a more secure and trustworthy network.

Decentralization and resilience: Opening the validator set to the wider community increases decentralization and enhances the network's resilience against malicious attacks. By distributing the validation power among a larger set of validators, the network becomes less susceptible to control by a single entity or a small group of validators.

Cons:

Potential for bad actors: Opening up the validator set does introduce the possibility of bad actors entering the network. Currently, the validator set consists of permissioned strategic partners who have undergone vetting. With open access, there is a higher chance of malicious validators being included. This risk needs to be carefully managed to ensure the network's security.

Technical resources and costs: Opening the validator set and supporting a larger number of validators would require additional technical resources and incur costs for Numbers Protocol. The infrastructure and support needed for an expanded validator set should be carefully considered to maintain a reliable and robust network.

In conclusion, opening up the validator set and implementing delegated PoS appears to have several benefits for Numbers Protocol. It aligns with the core values of community and decentralization, introduces delegation and rewards, increases utility and risk mitigation, and enhances network decentralization and resilience. However, the potential risks of bad actors and the technical requirements and costs should be thoroughly addressed. It is essential for the community to engage in open dialogue, share perspectives, and carefully consider the implications before making a final decision.

**Comment by Majorazors** [comment]
My take on this event of opening up the validator set and implementing a delegate PoS - Coming from a logical standpoint I think it should be put off for now pending a substantial amount of growth in the Numbers community and adequate education of the community members. 
Sitting on one of the cons outlined I can not stress enough the outcome or possible turn of events when something as important and delicate as this finds it ways in to the hands of bad actors. Especially in a case where the bad actors outnumber the good players. 
As much as this adds more utility for the $NUM token I think pending when community members and stakers are well informed and educated The Numbers partners should keep running it for the time being.

**Comment by 4djc** [comment]
It's really commendable that the team is looking forward to include the community in its validator roles for more decentralisation

As stated in the article, there are pros and cons. The delegated proof of stake is an adoptable idea but validator roles should be left for the Numbers partners to be able to keep track and do away with "bad actors"

Through the delegated PoS, community members will still have a voice through their staked tokens and will get to secure rewards for themselves.

Asides bad actors, the technological expertise is another issue the team should look into closely.
Do these community members have the necessary tools?
Are they conversant with validator roles?

These are questions that should be asked

**Comment by TylerDylan** [comment]
Based on the provided information, it seems that opening up the validator set and implementing delegated proof-of-stake (PoS) could be a favorable decision for Numbers. Here are the reasons supporting this stance:

Pros:

Alignment with core values: Opening the validator set and embracing decentralization aligns with the core values of Numbers Protocol, which prioritize community involvement and decentralization. It allows for greater participation and empowers community members to contribute to the network's validation process.

Delegation and rewards: Allowing NUM token holders to delegate their stake to validators of their choice introduces a mechanism for sharing in validation rewards. This incentivizes token holders to actively participate and stake their NUM tokens, adding utility to the token itself.

Increased utility and risk mitigation: Requiring the staking of NUM tokens and the possibility of slashing in the case of harmful behavior adds utility to the token. It also acts as a deterrent against malicious actions, as validators risk losing their staked tokens if they engage in harmful behavior. This can contribute to a more secure and trustworthy network.

Decentralization and resilience: Opening the validator set to the wider community increases decentralization and enhances the network's resilience against malicious attacks. By distributing the validation power among a larger set of validators, the network becomes less susceptible to control by a single entity or a small group of validators.

Cons:

Potential for bad actors: Opening up the validator set does introduce the possibility of bad actors entering the network. Currently, the validator set consists of permissioned strategic partners who have undergone vetting. With open access, there is a higher chance of malicious validators being included. This risk needs to be carefully managed to ensure the network's security.

Technical resources and costs: Opening the validator set and supporting a larger number of validators would require additional technical resources and incur costs for Numbers Protocol. The infrastructure and support needed for an expanded validator set should be carefully considered to maintain a reliable and robust network.

In conclusion, opening up the validator set and implementing delegated PoS appears to have several benefits for Numbers Protocol. It aligns with the core values of community and decentralization, introduces delegation and rewards, increases utility and risk mitigation, and enhances network decentralization and resilience. However, the potential risks of bad actors and the technical requirements and costs should be thoroughly addressed. It is essential for the community to engage in open dialogue, share perspectives, and carefully consider the implications before making a final decision.

**Comment by Mahabirgope** [comment]
Based on the information provided, it appears that implementing delegated proof-of-stake (PoS) and opening up the validator set could be advantageous for Numbers. Here are the reasons supporting this viewpoint:

Pros:

1. Alignment with core values: Embracing decentralization and allowing community participation aligns with Numbers Protocol's core values, emphasizing community involvement and decentralization. This enables greater engagement and empowers community members to contribute to the network's validation process.

2. Delegation and rewards: Allowing NUM token holders to delegate their stake to validators of their choice introduces a mechanism for sharing in validation rewards. This incentivizes token holders to actively participate and stake their NUM tokens, adding utility to the token itself.

3. Increased utility and risk mitigation: Requiring the staking of NUM tokens and the potential for slashing in the case of harmful behavior adds utility to the token. It also acts as a deterrent against malicious actions, as validators risk losing their staked tokens if they engage in harmful behavior. This contributes to a more secure and trustworthy network.

4. Decentralization and resilience: Opening the validator set to a broader community enhances decentralization and strengthens the network's resilience against malicious attacks. By distributing the validation power among a larger set of validators, the network becomes less vulnerable to control by a single entity or a small group of validators.

Cons:

1. Potential for bad actors: Opening up the validator set introduces the possibility of bad actors entering the network. Currently, the validator set consists of vetted permissioned strategic partners. With open access, the risk of including malicious validators increases. This risk needs to be managed effectively to ensure the network's security.

2. Technical resources and costs: Expanding the validator set and supporting a larger number of validators would require additional technical resources and incur costs for Numbers Protocol. It is crucial to carefully consider the infrastructure and support needed to maintain a reliable and robust network.

In conclusion, implementing delegated PoS and opening up the validator set appears to offer several benefits for Numbers Protocol. It aligns with core values, introduces delegation and rewards, enhances utility and risk mitigation, and strengthens network decentralization and resilience. However, it is essential to address the potential risks of bad actors and carefully consider the technical requirements and costs associated with such a change. Engaging in open dialogue, sharing perspectives, and thoroughly evaluating the implications are crucial steps before making a final decision.

**Comment by Mahabirgope** [comment]
Based on the available information, it appears that Numbers could greatly benefit from opening up its validator set and implementing delegated proof-of-stake (PoS). Here are the arguments supporting this viewpoint:

Advantages:

1. Consistent with core principles: By embracing decentralization and involving the community, Numbers Protocol can stay true to its fundamental values. This approach encourages wider participation and empowers community members to contribute to the validation process, fostering a sense of ownership.

2. Delegation and incentives: Allowing NUM token holders to delegate their stake to validators of their choosing introduces a rewarding mechanism. This incentivizes token holders to actively engage and stake their NUM tokens, thereby enhancing the value and utility of the token itself.

3. Enhanced utility and risk management: Requiring NUM token staking and the possibility of slashing in case of misconduct adds utility to the token. Moreover, it acts as a deterrent against malicious behavior since validators risk losing their staked tokens. This helps establish a more secure and trustworthy network.

4. Decentralization and resilience: By opening up the validator set to a broader community, Numbers Protocol can promote decentralization and bolster the network's resistance against malicious attacks. Distributing the validation power among numerous validators reduces the risk of control by a single entity or a small group.

Disadvantages:

1. Potential for malicious actors: Allowing open access to the validator set does introduce the risk of malicious actors infiltrating the network. Presently, the validator set consists of carefully vetted strategic partners. However, with open access, there is a higher probability of including validators with ill intent. This risk must be managed effectively to maintain network security.

2. Technical requirements and expenses: Expanding the validator set and supporting a larger number of validators necessitates additional technical resources and incurs costs for Numbers Protocol. Ensuring the network remains reliable and robust would require careful consideration of the infrastructure and support required.

To summarize, the advantages of opening up the validator set and implementing delegated PoS for Numbers Protocol are clear. It aligns with core principles, introduces delegation and incentives, enhances utility and risk management, and strengthens decentralization and resilience. However, the potential risks associated with malicious actors and the technical demands and costs should be addressed thoroughly. Engaging the community in open discussions, sharing perspectives, and considering the implications will be crucial before reaching a final decision.

**Comment by Nezukokamado11** [comment]
Expanding the validator set and implementing delegated proof-of-stake in Numbers Protocol has the potential to increase community participation, enhance decentralization, and foster trust in the network. The community review process aims to gather valuable insights and perspectives from the community, ultimately leading to informed decisions and a stronger ecosystem. Your engagement and input are greatly appreciated as we collectively explore the possibilities and implications of this proposal.

**Comment by 4djc** [comment]
Given the information at hand, it is advised that Numbers Protocol think about growing its validator network and putting delegated proof-of-stake (PoS) into place. Decentralization and community participation are two essential tenets of the protocol, therefore such a choice would be appropriate. By adopting this strategy, the protocol can gain a variety of advantages, including as increased community involvement, stake delegation, the sharing of validation rewards, greater token usefulness, decreased risks due to staking and slashing mechanisms, and improved network security. However, it is critical to take into account any potential disadvantages, such as the potential entry of bad actors into the network and the need for more technical resources and costs to maintain a bigger range of validators. It is crucial that the community do an investigation before making a decision.
It is critical that the community has an open and thorough conversation before making a choice in order to carefully weigh the benefits and drawbacks.

**Comment by TylerDylan** [comment]
Opening up the validator set and implementing delegated proof-of-stake (PoS) could be a favorable decision for Numbers Protocol. This aligns with their core values of community involvement and decentralization. The pros include greater community participation, delegation and rewards for token holders, increased utility and risk mitigation, as well as enhanced decentralization and resilience against attacks. However, there are potential risks of bad actors entering the network and the need for additional technical resources and costs. The community should engage in dialogue and consider the implications before making a final decision.

**Comment by Mahabirgope** [comment]
Opening up the validator set and implementing delegated proof-of-stake (PoS) for Numbers Protocol has several potential benefits. It aligns with the core values of community involvement and decentralization, allowing for greater participation and empowering token holders to contribute to the network's validation process. Delegation introduces rewards and incentivizes token holders to stake their NUM tokens, adding utility to the token. It also enhances network security and trustworthiness by requiring staking and implementing potential slashing for harmful behavior. Furthermore, expanding the validator set increases decentralization and strengthens the network's resilience against attacks. However, careful management of the risks associated with bad actors and consideration of technical requirements and costs are essential. Engaging the community, conducting thorough evaluations, and fostering open dialogue are vital in making an informed decision about implementing delegated PoS and expanding the validator set for Numbers Protocol.

**Comment by Desure01** [comment]
In my own review: Expanding the validator set and implementing delegated proof-of-stake in Numbers Protocol has the full fundamentals to add value to increase the community participation! This move can empower NUM token holders , with the ability of contribution all can be processed accordingly… 

Meanwhile The aims for the review is to provide valuable perspectives opinion from the community, of which the decisions of the community can also be a great help also an healthy ecosystem.

The Number Team always know the heart of the community!

**Comment by mdrizwan-9** [comment]
In my perspective, the decision to open up the validator set and implement delegated PoS  in Numbers Protocol presents both advantages and disadvantages. On the positive side, it aligns with the core values of community engagement and decentralization, granting the community a sense of empowerment and expanding the utility of the NUM token. Furthermore, it can bolster network security by introducing a broader range of validators and diversifying their origins. 

However, there are noteworthy considerations, including the potential entry of malicious individuals into the network, the irreversible nature of this shift, and the necessity for technical proficiency and adequate resources. 

Meticulous assessment and risk analysis are imperative to strike a harmonious balance between community participation and network integrity. I think the team should go forward and open up the validator set to the public.

**Comment by Nezukokamado11** [comment]
According to the data available, Numbers Protocol should consider increasing its validator network and implementing delegated proof-of-stake (PoS). This choice supports the protocol's essential concepts of decentralisation and community engagement. This technique has several advantages, including increased community involvement, stake delegation, shared validation rewards, greater token utility, risk reduction through staking and cutting, and improved network security. However, there may be drawbacks to consider, such as the ability of bad actors to penetrate the network and the need for additional technical resources and costs to support a larger number of validators. Before making a decision, the community must hold an open discussion and carefully assess the benefits and drawbacks.

**Comment by Defiareana** [comment]
In my viewpoint, the positive effects of decentralizing the validator set inside the Numbers Blockchain much outweigh the costs to the ecosystem. Numbers Protocol might make use of community involvement and decentralization by making the validator set available to a larger community.

By enabling them to actively participate in the validation process and share in the rewards, this action may give NUM token holders more influence. This not only boosts the NUM token's usefulness but also encourages new users to hold NUM and participate in the network, creating a more open and robust blockchain network.

**Comment by emmerson121** [comment]
Numbers has been actively working towards a decentralized community. After going through the pros and cons, it is pertinent to commend that the pros are actually going to add more value and make the $NUM token stand-out but having evaluating the consequences of opening up the mainnet validators to the community, I would say everything should be maintained the way it is currently.

Since opening up the validator set is irreversible once it is done, it calls for concern if it ends up not being sustainable and since it is will cost revenues from Numbers Protocol, it'll be a waste if it ends up not living up to expectations.

**Comment by valirvan** [comment]
I am very impressed by  Numbers initiative to increase network resilience and security by introducing delegated proof-of-stake (PoS). This means that anyone in the community can become a validator and help maintain the network’s integrity. I think this is a great way to prevent centralization and protect the network from malicious attacks. It also makes the protocol more efficient by speeding up block confirmation times.

Another benefit of delegated PoS is that it involves NUM holders in the consensus process. They can delegate NUM tokens to validators they trust, which creates a sense of accountability and responsibility among the validators. They have a stake in the network’s stability and security, so they will act in the best interest of the community. This also builds trust and engagement among the token holders, who can have a say in how the network is run. I think this will be a good move to take action.

**Comment by TylerDylan** [comment]
The provided information suggests that Numbers Protocol should consider opening up the validator set and implementing delegated proof-of-stake (PoS). This decision aligns with the protocol's core values of community involvement and decentralization. By allowing token holders to delegate their stake to validators, it introduces a reward mechanism that incentivizes participation and adds utility to the token. Requiring staking and the possibility of slashing mitigates risks and enhances network security. Additionally, decentralization improves resilience against attacks. However, there are potential drawbacks, such as the risk of bad actors entering the network and the need for additional technical resources and costs to support a larger validator set. These factors should be carefully managed and considered through open dialogue and community engagement before a final decision is made.

**Comment by WEB3Queen01** [comment]
As I thoroughly assess the pros and cons, the decision to open the validator set should consider the long-term goals, security requirements, and level of community participation desired. Increasing decentralization and attack resilience through a larger set of validators can help protect against malicious attacks. Just It is crucial to strike a balance between decentralization and maintaining a secure and reliable network.

**Comment by avie018** [comment]
By decentralizing the validator set, I see the Numbers Protocol aiming to achieve several potential benefits:


Increased Community Participation: Opening up the validator set allows a broader range of community members to participate in the validation process. This inclusivity promotes community engagement and ownership of the network.


Enhanced Security and Resilience: A decentralized validator set reduces the risk of a single point of failure or control. It enhances the network's security by distributing validation responsibilities among multiple validators, making it more resistant to attacks and manipulation.


Diversification of Validators: Allowing new validators to join the network brings in fresh perspectives and expertise. It fosters innovation and ensures a more diverse and robust validator set, further strengthening the network's integrity.


Competitive Validation Rewards: With delegated proof-of-stake, NUM token holders can delegate their stake to validators of their choice. This mechanism encourages healthy competition among validators to attract delegations, leading to improved services and potentially higher validation rewards for token holders.


Decentralized Governance: A decentralized validator set aligns with the principles of decentralized governance. It provides an opportunity for a wider range of stakeholders to have a say in decision-making processes and protocol upgrades, leading to more inclusive and representative governance.


And in my conclusion, decentralizing the validator set aligns with the community and decentralization values of the Numbers Protocol. It has the potential to increase community participation, enhance security, foster innovation, and promote decentralized governance. However, careful consideration must be given to addressing the risks associated with bad actors, technical requirements, monitoring, and governance complexity. Through thoughtful planning and active community engagement, the Numbers Protocol can pave the way for a more decentralized and robust network.

**Comment by Tahira19** [comment]
In my perspective, adopting delegated proof of stake while expanding the Numbers community is viewed favourably as being useful in terms of creating interpersonal ties and building trust among community members. But crucial decisions must be taken.

The following are a few benefits of accessing the validator set:

Decentralization: The network can become more decentralised by letting more users to become validators or assign tokens to other users. By distributing authority, reaching consensus, and decision-making across a wider range of stakeholders, the network's security and resilience are improved.

Broader Participation: Expanding the validator set and enabling token delegation can promote greater token holder participation. It makes it possible for people with less technical expertise or resources to still contribute to the consensus-building process and actively take part in network security.

Reduced Entry Barrier: The entry barrier to taking part in consensus is reduced by allowing more individuals to become validators or delegate their tokens. This encourages a more inclusive and diversified network by making it easier for smaller stakeholders to participate.

Network security can be improved with the use of a decentralised validator set. A larger number of validators reduces the possibility of collusion or malicious activity, making it more challenging for any one entity to jeopardise the integrity of the network.

Cons of Opening up the Validator Set and Implementing Delegated PoS:

Governance Complexity: Delegated PoS introduces additional complexity to the governance process. Coordinating voting and decision-making among numerous validators and their delegated token holders can be challenging and may require the development of effective governance mechanisms.

Ultimately, the decision to open up the validator set and implement delegated PoS should be based on a thorough evaluation of the specific needs, goals, and dynamics of NumbersDAO, as well as input and consensus from the community members.

**Comment by Mahabirgope** [comment]
Embracing an open validator set and implementing delegated proof-of-stake (PoS) could be a game-changer for Numbers Protocol. It perfectly aligns with their core values of community engagement and decentralization. The benefits A thriving community, rewarding token holders through delegation, mitigating risks, boosting utility, and fortifying against attacks. Of course, there are challenges to tackle, like potential bad actors and increased technical demands. Let's foster an open dialogue within the community to carefully evaluate the implications. Exciting times ahead

**Comment by 12divine** [comment]
Based on the various benefits shown in the article below, I believe numbers should approve and implement the delegated POS technology, it's similar to what ethereum uses, and will go a long way to better the chain! So it's a yes from me.

**Comment by rahulkr2503** [comment]
I personally feel Numbers Protocol will get many benefits from opening up its validator set and implementing delegated PoS. Opening up the validator set & implementing delegated PoS could be a best decision for Numbers Protocol. 
It will bring community involvement and network will become more decentralized. Allowing NUM token holders to delegate their stake to validators of their choice introduces a mechanism for sharing in validation rewards. This incentivizes token holders to actively participate and stake their NUM tokens, adding utility to the token itself.
However, there are certain risk which team must look before implementing .
There is potential risks of bad players entering the network and the need for additional technical resources and costs. By opening the validator set, this opens the door to higher chances of bad actors. Once implemented this action is irreversible, as the validator set will become permissionless and there is no way to remove all new validators.
NUM team should consider the implications before making a final decision.
All i know that Numbers Protcol team is well experienced and will surely do what is best for NUM holders.

**Comment by Giftino** [comment]
Numbers Protocol has been actively dedicated to establishing a decentralized community. Upon thorough examination of the pros and cons, it is essential to acknowledge that the benefits will significantly enhance the value and distinctiveness of the $NUM token. However, after evaluating the potential ramifications of granting access to the mainnet validators for the community, it is recommended to maintain the current setup.

The irreversible nature of opening up the validator set raises concerns about sustainability. Considering the potential loss of revenues for Numbers Protocol, it would be unfortunate if the decision does not meet expectations and proves to be futile.

**Comment by samxonite** [comment]
Increasing the validator set of a blockchain can have several advantages and disadvantages.
Pros: A larger validator set enhances the security of the blockchain, Increasing the validator set can enhance the consensus mechanism of the blockchain. With more validators, the network can achieve a more reliable and accurate consensus on the state of the blockchain, ensuring that only valid transactions are included.

Cons: Increasing the validator set can complicate the governance of the Numbers network. Decision-making processes, protocol upgrades, agreement among a larger number of validators, potentially leading to slower decision-making and implementation.

considering these pros and cons should be noted in the context of the Numbers goals at this phase. including the desired level of decentralization, security requirements, scalability goals.

**Comment by Aniel756** [comment]
By the vision of Numbers Protocol is to build a strong network that not only empower its participants but also enable secured transactions and trust. 

Opening the validators set will see to a lot of advantages to the Numbers protocol which are not limited to:

1. Decentralisation and Alignment with core values: opening up the validators set will increase decentralisation and will also align with the core values of Numbers Protocol.

2. Increased utility of NUM tokens: opening the validator set will increase the utility of NUM tokens and also community participation, it will enhance token utility and network security.

There are more advantages that comes with opening the validators set but it will also open up the network to some disadvantages which are:

1. Making the network a potential for bad actors: opening up the validator set means there won't be a limit as to who will enter the network. And this can open the network to bad actors who want to bring down the core values of Numbers Protocol.

In conclusion, opening up the validator set will have a lot of benefits for Numbers Protocol. It will bring about decentralisation, strengthen Numbers core values and bring additional rewards. But the issue that needs to be addressed  are the risk of bad actors getting involved and the cost of revenue being on a high side. 

If these are addressed efficiently, then opening the validators set will be the next step to bring the vision of Numbers protocol closer to the community.

**Comment by fredlex** [comment]
Implementing the proof of stake POS For delegates, opening of the validator set to a wider Community helps to build the interrelationship among community members hence creating an improved utility for the $NUM token since this makes community members benefit from the profit of staking being us closers to everything decentralization stands for.

Proper measures should be taken to eradicate any form of bad actions from validators to keep the community completely secured. ..

**Comment by Radikapratama** [comment]
> ### Topic: Should Numbers look to open up the validator set and implement delegated PoS?
> ### Overview
> Today, we embark on a journey of exploration and discussion surrounding an important decision that lies at the heart of our decentralized ecosystem. We are here to deliberate on the question of whether Numbers should open up its validator set to the wider community.
> 
> At Numbers Protocol, we deeply value our community and the principles of decentralization. Our shared vision is to build a robust and resilient network that empowers its participants, enables secure transactions, and fosters trust. The selection of validators plays a critical role in achieving these goals within a proof-of-stake blockchain. Currently, our validator slots are permissions and limited to Numbers partners. However, we believe it is time to open up the discussion and consider the advantages and challenges of embracing a more inclusive approach.
> 
> This article will examine the pros and cons of allowing access to the current permission Numbers blockchain validator set and implementing a delegated proof-of-stake (PoS) mechanism, similar to other prominent layer-1 blockchains. We aim to provide you with an impartial analysis that equips you with the necessary information to understand the dialogue that will take place during the upcoming DAO meeting. Remember, this article is not meant to sway your opinion but rather to foster dialogue and broaden our collective understanding.
> 
> So, let us delve into the intricacies of decentralizing the validator set and explore its potential impact on our community, protocol, and Numbers' future.
> 
> ### Decentralizing the validator set
> Before diving into the discussion around decentralizing the validator set, first, we will outline what a validators role is within a proof-of-stake blockchain:
> 
> _In Proof of Stake blockchains, validators are selected to produce the next block based on their stake in the network token. These validators will stake some of NUM tokens and run software that allows them to produce and approve blocks. Although often designed with random functions to prevent a front-running consensus, a larger amount staked by a validator could give them a higher chance of producing the next block. Proposed blocks by validators are then propagated to the rest of the set, who verify and add the approved block to the blockchain._
> 
> In the current implementation of the Numbers Blockchain, these validator slots are permissions and only accessible to Numbers partners.
> 
> Community and decentralization are core values of Numbers Protocol, and with these values in mind, we have decided to open up the discussion around allowing access to the currently permissioned Numbers blockchain validator set and implementing a delegated proof-of-stake (PoS) mechanism, similar to other popular layer-1 blockchains.
> 
> This community article aims to outline the pros and cons of opening the validator set to the community:
> 
> **Pros**
> 
> * Opening the validator set is in-line with the core values of Numbers Protocol, which are community and decentralization
> * Allows for NUM stakers to delegate their stake to a validator of their choice to share in validation rewards
> * Adds additional utility for the NUM token, as this token will be required to be staked and risks slashing in the case of harmful behavior
> * Allows for additional rewards for those community members who want to validate the Numbers Blockchain
> * Increases decentralization and attack resilience in the event of a malicious validator attacking the network
> 
> **Cons**
> 
> * Current validators are permissioned strategic partners of Numbers Protocol, which includes AVA. By opening the validator set, this opens the door to higher chances of bad actors. Once implemented this action is irreversible, as the validator set will become permissionless and there is no way to remove all new validators.
> * Opening the validator set will require technical resources and costs from Numbers Protocol revenues
> * Running a validator will require technical expertise on the community members part to run effectively and without downtime in addition to a sizeable stake in NUM
> 
> Numbers is actively working with its partner network to assign more utility to the NUM token and work towards a more decentralized future. This article aims at opening the discussion between the community if a decentralized validator set is something that would like to be explored.
> 
> ### Conclusion
> Ultimately, the decision to decentralize the validator set within Numbers Protocol is one of great significance and should not be taken lightly. As we assess the advantages and challenges of such a step, we must remain steadfast in our commitment to community and decentralization, which lie at the core of our protocol.
> 
> By opening up the validator set, we can embrace a more inclusive approach that empowers NUM stakers to delegate their stake, share in validation rewards, and contribute to the robustness of our blockchain. This move enhances the utility of the NUM token and fortifies the network's resilience against malicious attacks. However, we must be mindful of the additional risks that come with decentralization, such as potential bad actors and the technical resources required to support a larger validator set.
> 
> Numbers Protocol remains neutral in this discussion. We encourage all stakeholders to engage in open dialogue and share their perspectives.
> 
> Together, let us shape the future of Numbers Protocol as we work towards a more decentralized and inclusive ecosystem driven by the collective wisdom of our community. Your voice matters, and we eagerly anticipate the outcome of this crucial decision.
> 
> Thank you for being an integral part of the Numbers Protocol journey!
> 
> ### Entering Community Review
> Just like previous DAO meetings topics, this Numbers Governance Proposal will undergo community review until June 20. During this time, we encourage the Numbers community to share your thoughts and concerns on GitHub issues and discuss this proposal openly on our many social channels. We look forward to hearing your feedback!
> 
> ┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1203248692432071/1204757790001833) by [Unito](https://www.unito.io)

Good project and nice project and I hope long live for this project.. 
LFG

**Comment by a2tim** [comment]
In my opinion.. opening the validator set to a wider Numbers community while implementing delegated proof of stake is seen from a positive light as being beneficial improving inter relationship and trust building among members of the community. However, critical decisions needs to be made.

Here are few advantages of opening the validator set:

Opening the validator set is in-line with the core values of Numbers Protocol, which are community and decentralization
Allows for NUM stakers to delegate their stake to a validator of their choice to share in validation rewards
2. Staking reward: Adds additional utility for the NUM token, as this token will be required to be staked and risks slashing in the case of harmful behavior
Allows for additional rewards for those community members who want to validate the Numbers Blockchain

Cons..
Opening of the validator set would require a robust technical and financial input to present and efficient staking ecosystem

Similarly an expected outcome of potential bad actors should not be ruled out as this may in turn reduce trust level..

**Comment by nzubezion** [comment]
Yes, opening up the validator set and implementing delegated proof-of-stake (PoS) is a positive step for Numbers Protocol. It aligns with Numbers' core values of community and decentralization, empowering NUM stakers to actively participate in the validation process and share in rewards. This move enhances the utility of the NUM token and strengthens the security and resilience of the blockchain network.

**Comment by Zeecok** [comment]
Goood

**Comment by DeFiDebbie** [comment]
I'm sure you're aware that, unlike the proof of stake, dPoS encourages scalability due to the limited number of block producers. In PoS, the mechanism only favours the whales. If you're talking about incentivisation, you should have your community in view. There should be no barrier of entry but rather a floor to prove themselves worthy of being a Numbers Validator.

**Comment by Mohitsid0** [comment]
Numbers Protocol is advised to explore the option of opening up its validator set and incorporating delegated proof-of-stake (PoS) as suggested in the available information. This choice aligns well with the protocol's fundamental principles of community engagement and decentralization. There are several benefits associated with this approach, including increased community involvement, stake delegation, shared validation rewards, enhanced token utility, risk management through staking and slashing, and bolstered network security. However, it is essential to consider potential drawbacks, such as the potential entry of malicious actors into the network and the requirement for additional technical resources and expenses to support a larger group of validators. Prior to reaching a final decision, it is crucial for the community to engage in transparent discussions and carefully assess the advantages and disadvantages of this proposal.

**Comment by Desure01** [comment]
Meanwhile based on what I saw in the article.

I believe numbers team should implement the delegated POS technology, the Numbers community and decentralization, empowering NUM stakers to actively participate in the validation process and share in rewards. This is a utility of the NUM token.

**Comment by BTComrade** [comment]
with a vision and a plan already set in place increasing the validator set would bring great benefits.
Here are some pros and cons to consider
Pros of Increasing the Validator Set: A larger validator set increases the level of decentralization in the blockchain network. With more validators, power and decision-making authority are distributed among a larger group of participants, reducing the risk of a single entity or group gaining control over the network.
A larger validator set would also enhance the security of the blockchain.

Cons of Increasing the Validator Set: As the validator set grows, coordinating decision-making and consensus among a larger group of participants becomes more challenging. It may require additional mechanisms and protocols to manage communication, voting, and governance processes effectively.

**Comment by ritoAryan** [comment]
Expanding the validator set and introducing delegated Proof of Stake (PoS) offer numerous advantages for Numbers. By increasing the size of the validator set, Numbers can enhance decentralization, enabling more participants to contribute to the network's consensus. This, in turn, improves the overall security and resilience of the blockchain.

The implementation of delegated PoS allows token holders to delegate their stake to trusted validators who can represent their interests and participate in the consensus process on their behalf. This approach encourages engagement and involvement from a wider community, including those with limited technical expertise or resources to operate their own validators. It also provides incentives for token holders to stake their assets, which contributes to the network's security and stability.

**Comment by Dharam0965** [comment]
Numbers should consider opening up the validator set and implementing delegated Proof of Stake (PoS) for the following reasons:

Advantages:

1. Community engagement and decentralization: By allowing more participants to become validators, Numbers can involve a larger community in the network's consensus process, leading to increased decentralization.

2. Delegation for rewards: Token holders can delegate their stake to validators and earn rewards in return. This incentivizes participation and provides an opportunity for token holders to benefit from their holdings.

3. Enhanced utility and risk management: Opening up the validator set can increase the overall utility of the network by allowing more transactions to be processed. Additionally, delegated PoS can help mitigate the risk associated with a single validator holding a significant portion of the stake.

4. Improved decentralization and network resilience: The inclusion of a larger number of validators improves the decentralization of the network, making it less prone to manipulation or control by a single entity. This, in turn, enhances the resilience of the network.

Disadvantages:

1. Potential for malicious actors: Opening up the validator set introduces the possibility of bad actors entering the network. This risk should be carefully managed through robust vetting processes and security measures.

2. Additional technical resources and costs: Implementing delegated PoS requires technical resources and infrastructure to support the increased number of validators. This can lead to additional costs and complexities for the network.

In summary, the proposed changes of opening up the validator set and implementing delegated PoS offer numerous benefits, including community participation, rewards, and improved network security. However, they also bring certain risks that need to be carefully addressed and managed.

**Comment by luckymod4** [comment]
Implementing Proof of Stake (PoS) and opening the validator set will greatly enhance the utility of the Numbers Protocol native token, fostering increased community trust and driving growth within the Numbers ecosystem. This move aligns with our core values and holds promising potential. While there may be concerns, such as the risk of bad actors and the requirement for financial and technical involvement, we can address them by implementing robust security protection technology and establishing strict criteria for validator recognition. Together, we can create a seamless environment while keeping malicious actors at bay. #NumbersProtocol #PoS #CommunityTrust

**Comment by NuelDesigns99** [comment]
It introduces the concept of decentralizing the validator set in Numbers Protocol and outlines the advantages and disadvantages of such a move. It acknowledges the core values of community and decentralization and highlights the potential benefits of opening up the validator set to the wider community.

The pros presented in the article include aligning with the protocol's core values, empowering NUM token holders, adding utility to the token, and enhancing decentralization and attack resilience. On the other hand, the cons discussed involve the risks associated with bad actors, the technical requirements and costs, and the need for expertise and a significant stake in NUM.

The write-up concludes by emphasizing the importance of the decision and the commitment to community and decentralization. It encourages further discussion and consideration of the potential impact.

Overall, the write-up provides a balanced view of the topic, presenting both the advantages and challenges associated with opening up the validator set. It aims to inform readers and stimulate dialogue within the Numbers Protocol community.

**Comment by eserboo** [comment]
Am sure you know that unlike the proof of stake, dPoS encourages scalability.

In PoS, the mechanism only favours the whales. If you're talking about incentivisation, the community view is very important.

**Comment by CaptainJackSparrow09** [comment]
I have been using Capture App since three months now. Overall experience on the application is really amazing. From launching the app to converting a picture into NFT in just few clicks. 

However, one issue that I would like to address here is Capture App isn't all devices friendly. Personally I have two devices one is Realme and another is Vivo. On Realme Capture App works fine while on Vivo Capture camera doesn't support at all which dilutes my motivation to navigate Capture App sometimes. I would be very happy if Capture App fixed this issue permanently. 

I believe, it will help to on-board million more users.

**Comment by cryptomate7863** [comment]
I. Introduction


Explanation of the Numbers Protocol's plan to open up its validator set and implement a delegated proof-of-stake mechanism

Brief overview of how proof-of-stake blockchains work

II. Pros of Opening up the Validator Set to the Community


Increased decentralization and community involvement

Increased attack resilience

Ability for token holders to delegate their stake and earn rewards

Potential for a more efficient and cost-effective consensus mechanism

III. Cons of Opening up the Validator Set to the Community


Risk of centralization if a few large stakeholders dominate the validator set

Potential for vote buying and other forms of collusion

Increased complexity in managing the validator set and ensuring its security

Possibility for decreased network performance if inexperienced validators are selected

IV. Conclusion


Summary of the pros and cons of opening up the validator set to the community

Discussion of how the Numbers Protocol can mitigate the risks and maximize the benefits of this decision

Final thoughts on the potential impact of this decision on the Numbers Protocol and the wider blockchain ecosystem

**Comment by PhaeTee** [comment]
The roles of validators is an essential one in every project and i believe if the Numbers community members are granted the opportunities as well to become validators, it'll actively promote trust and also give a chance to lovers and true believers of the protocol and those who were not opportuned to be among early partners of Numbers to invest seeing the potentials of the project and also the rewards that comes with it.
 Everybody loves a good reward system i must say

**Comment by Kristeeyo** [comment]
The implementation of the delegated Proof of Stake (PoS) consensus mechanism is beneficial to Numbers Protocol as well as the community members 

Greater interaction of the members with the Numbers ecosystem will be achieved thereby also increasing the utility of the $NUM token and maintaining the core value of Numbers - decentralisation and community

However, the cons can't be overlooked too as bad actors can be a major cause of concern. Another concern may be in the field of technicality.

Regardless, the advantages outweighs the disadvantages. This new development will have a positive impact on the Numbers Ecosystem.

**Comment by RisingStar05** [comment]
To be honest, there is nothing that doesn't have disadvantages attached to them. Choosing dPOS as a new staking mechanism gives more weight to NUM and actively involves the community members in securing the blockchain. Not only that but being able to provide a great reward system such as this is highly commendable.

**Comment by ElieTrades** [comment]
Implementing the delegated proof of Stake consensus mechanism (dPOS) is a huge step towards adopting complete decentralization in Numbers protocol. 

The chance of bad actors trying to game the system might be slim if the community is given the time to fully get familiar with who they will vote to become validators. Overall, I believe you should rely on community votes on this topic.

**Comment by qqorq** [comment]
As Mayasharmark said, The topic is ideal for community engagement. Implementing delegated PoS for Numbers involves weighing advantages (enhanced security, community participation) and disadvantages (slower decision-making, potential troublemakers). 

I also believe the community must carefully evaluate these aspects to make an informed decision for the Numbers protocol.

**Comment by Markie07** [comment]
It is great that you open this topic. As a read this topic, the proposal to open up the validator set in Numbers Protocol raises an important question about community participation and decentralization. It acknowledges the significance of validators in achieving the protocol's goals and aims to explore the advantages and challenges of embracing a more inclusive approach.

The article presents a balanced analysis of the pros and cons associated with this proposal. On the positive side, opening up the validator set aligns with the core values of Numbers Protocol, emphasizing community involvement and decentralization. It allows NUM token holders to delegate their stake and receive validation rewards, which enhances the utility of the token. Moreover, it promotes increased decentralization and resilience against malicious attacks.

However, the proposal also acknowledges potential challenges. Opening the validator set may introduce the possibility of bad actors, as the current permissioned validators would be joined by new ones. It requires technical resources and costs from Numbers Protocol, and running a validator necessitates technical expertise and a significant stake in NUM.

The conclusion emphasizes the gravity of the decision and underscores the commitment to community and decentralization. It encourages stakeholders to engage in open dialogue and share their perspectives during the community review phase.

Overall, the proposal shows a thoughtful consideration of the benefits and drawbacks of opening up the validator set. It reflects the commitment of Numbers Protocol to its community and seeks to make an informed decision in line with the protocol's goals.

**Comment by Bikso1** [comment]
Expanding the validator set and implementing delegated PoS can improve decentralization, security, participation, and efficiency in blockchain projects. By allowing more participants to become validators, the network becomes more diverse, making it harder for malicious actors to exert undue influence. This also increases token holders' participation in securing transactions and earning rewards. However, it is crucial to consider potential challenges and risks, such as ensuring the reliability and trustworthiness of validators and reducing centralization. The decision to implement these changes should be carefully evaluated based on the blockchain project's context, goals, and characteristics.
